pages/linux/nmcli-general.md

```diff
@@ -0,0 +1,29 @@
+# nmcli general
+
+> Manage general settings of NetworkManager.
+> This subcommand can also be called with `nmcli g`.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+
+- Show the general status of NetworkManager:
+
+`nmcli general`
+
+- Show the hostname of the current device:
+
+`nmcli general hostname`
+
+- Change the hostname of the current device:
```

Comment:
```suggestion
- Change a hostname of the current device:
```

Reply:
`a` indicates that there is more than 1 hostname

Reply:
Thanks! I always thought that `a` just used when it doesn't matter what we are referring to but it seems it also used when there are more than one object in some context. In the context of some PC there is just one hostname. Now I hopefully understand why `the` is used here.